pylint_odoo/test/main.py

```diff
@@ -9,6 +9,7 @@
 
 
 EXPECTED_ERRORS = 52
+EXPECTED_ERRORS += 2  # Errors due to C8106: Wrong version format
```

Comment:
:o) Ok, but at some point (when there is no active PR), we should reset that to a single line.

Let's merge it and move forward.